ffv1.md

```diff
@@ -808,10 +816,26 @@ If state_transition_delta is not present in the FFV1 bitstream, all Range coder
 |-------|:--------------------------------|:--------------------------------|:--------------------------------|
 | 0     | YCbCr                           | No Pixel transformation         | plane then line                 |
 | 1     | RGB                             | JPEG2000-RCT                    | line then plane                 |
+| 2     | Color Filter Array              | JPEG2000-RCT + G difference     | line then plane                 |
 | Other | reserved for future use         | reserved for future use         | reserved for future use         |
 
 Restrictions:  
 If `colorspace_type` is 1, then `chroma_planes` MUST be 1, `log2_h_chroma_subsample` MUST be 0, and `log2_v_chroma_subsample` MUST be 0.  
+If `colorspace_type` is 2, then `chroma_planes` MUST be 1, `log2_h_chroma_subsample` MUST be 0, and `log2_v_chroma_subsample` MUST be 0, transparency MUST be 1.  
+
+### cfa_pattern
+
+`cfa_pattern` indicates the actual color filter array geometric pattern of the image sensor used to capture the single sensor color image.  
+The pattern is provided per line top to bottom, and for each line left to right.  
+
+|value  | color                           |
+|-------|:--------------------------------|
+|0      |   red                           |
+|1      |   green                         |
+|2      |   blue                          |
+
+Restrictions:  
+At least 1 component of each color must be present.  
```

Comment:
MUST

Reply:
Could add more restrictions, for instance it seems that the cfa array can not contain more than 4 values.

Reply:
uppercase and added more text about the restriction (4 values, width and height of 2)